src/scripts.js

```diff
@@ -36,33 +36,39 @@ const filterSection = document.querySelector('.filter-section-js');
 const mainView = document.querySelector('.main-view-container-js');
 const searchRecipesButton = document.getElementById('searchRecipes');
 const filterButton = document.getElementById('filterButton');
-const homeButton = document.querySelector('.home-button');
+const whatsCookin = document.querySelector('h1');
 const searchBar = document.getElementById('searchInput');
 const filterTags = document.querySelectorAll('.tag');
 const favoritePageButton = document.getElementById('favoritesPage');
 const addToCookbookButton = document.querySelector('.cookbook-button-js')
 const cookbookPageButton = document.querySelector('.cookbook-page-button-js');
 const singleViewFavoriteButton = document.querySelector('.single-view-favorite-button-js');
-const pageTitleButton = document.querySelector('.page-title-js');
+const pageTitle = document.querySelector('.page-title-js');
 
 // EVENT LISTENERS
 searchRecipesButton.addEventListener('click', () => {
   searchAllRecipes(recipeRepository.recipeData);
 });
+searchBar.addEventListener('keyup', function(e) {
```

Comment:
I'm glad we could add this feature. A small change that makes it way more user friendly!